framework/src/modules/network/network.js

```diff
@@ -61,16 +61,16 @@ module.exports = class Network {
 			await this.channel.invoke('app:getApplicationState')
 		);
 
-		const seedPeers = await lookupPeersIPs(this.options.list, true);
-		const blacklistedPeers = this.options.access.blackList
+		const seedPeers = await lookupPeersIPs(this.options.seedPeers, true);
+		const blacklistedPeers = this.options.blacklistedPeers
 			? this.options.access.blackList.map(peer => ({
 					ipAddress: peer.ip,
 					wsPort: peer.wsPort,
 			  }))
 			: [];
 		const p2pConfig = {
 			nodeInfo: initialNodeInfo,
-			hostAddress: this.options.address,
+			hostAddress: this.options.hostAddress,
```

Comment:
Seems you forget to revert this change; 

```suggestion
			hostAddress: this.options.address,
```